manifests/kots-config.yaml

```diff
@@ -356,7 +345,7 @@ spec:
           when: '{{repl and (ConfigOptionEquals "accessToCartoModeK8s" "accessToCartoModeK8sDefault") (eq Distribution "gke") }}'
           title: Global static IP Address name
           help_text: |-
-            Bring your static IP name. We'll link this IP to the External Load Balancer
```

Comment:
Please specify the name (not address) of a reserved external IP address in GCP to assign to the installation. If left blank, a random URL will be generated.